src/features/store.ts

```diff
@@ -53,4 +56,11 @@ export const useStore = create<AppState>()((set) => ({
   setIsSenderNftOwner: (isSenderNftOwner) => {
     set((state) => ({ balances: { ...state.balances, isSenderNftOwner } }));
   },
-}));
+});
+
+export const useStore = create<AppState>()(
+  persist(persistState, {
+    name: 'app-state',
+    storage: createJSONStorage(() => localStorage),
+  }),
+);
```

Comment:
A few items here:
1. Can you restructure the code to be organized like here: https://github.com/hyperlane-xyz/hyperlane-explorer/blob/a3e422016988fba77d74dc25ca8e26c9eb1c0ff7/src/store.ts#L37
2. Please add a `partialize` option to only persist the transfers
3. Is the `storage` field necessary? I thought it used JSON by default?

Reply:
1) Sure
2) Ok
3) Yeah I missed that, that is optional field